Userland/Games/Chess/ChessWidget.cpp

```diff
@@ -634,46 +634,49 @@ bool ChessWidget::export_pgn(StringView export_path) const
     }
     auto& file = *file_or_error.value();
 
+    StringBuilder builder;
     // Tag Pair Section
-    file.write("[Event \"Casual Game\"]\n");
-    file.write("[Site \"SerenityOS Chess\"]\n");
-    file.write(String::formatted("[Date \"{}\"]\n", Core::DateTime::now().to_string("%Y.%m.%d")));
-    file.write("[Round \"1\"]\n");
+    builder.append("[Event \"Casual Game\"]\n");
+    builder.append("[Event \"Casual Game\"]\n");
```

Comment:
Possible typo? Before this commit, this line was output only once.

Reply:
Certainly is!